packages/db/README.md

```diff
@@ -0,0 +1,46 @@
+# DB
+
+## Getting Started
```

Comment:
We need to update the main README pointing to the instruction in the single packages. Given that we still not have for the API, this can probably be a follow up PR